source-control/pull-request-templates/azure-devops/azure-devops-feature-pr-template.md

```diff
@@ -0,0 +1,35 @@
+# [Work Item](./link-to-the-work-item)
+
+## PR Checklist
+
+---
+
+<!-- Use the check list below to ensure your branch is ready for PR.  If the item is not applicable leave it blank. -->
+
+- [ ] I have updated the documentation accordingly.
+- [ ] I have added tests to cover my changes.
+- [ ] All new and existing tests passed.
+- [ ] My code follows the code style of this project.
+- [ ] I ran lint checks locally prior to submission.
+- [ ] Have you checked to ensure there aren't other open Pull Requests for the same update/change?
```

Comment:
Suggest changing to 1st party form so it's consistent with the other.  For example, "I have checked..." instead of "Have you checked...".  This also changes it from a question to a statement.

Reply:
good catch